### PR TITLE
Change default route to reencrypt for gateway

### PR DESCRIPTION
**Pull request by rubenvp8510**

### CODE REVIEW

File: internal/webhooks/tempostack_webhook.go [hunk 1]
```diff
@@ -39,7 +39,7 @@ var (
 type TempoStackWebhook struct {
 }
 
-const defaultRouteGatewayTLSTermination = v1alpha1.TLSRouteTerminationTypePassthrough
+const defaultRouteGatewayTLSTermination = v1alpha1.TLSRouteTerminationTypeReencrypt
```
Comment (frzifus) [inline internal/webhooks/tempostack_webhook.go:42]: Do we need a changelog for this?

Comment (rubenvp8510) [reply internal/webhooks/tempostack_webhook.go:42]: I think so, I mean is an small change, but may be the users wanted to be aware?

File: .chloggen/reencrypt_gw_default.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
```
Comment (frzifus) [inline .chloggen/reencrypt_gw_default.yaml:2]: Is it maybe breaking? Since some certs will change.

**Review by frzifus — APPROVED** [review]
I am unsure with the change type, but aside from that lgtm!

**Comment by frzifus** [comment]
Ahh, monolithic-receivers-tls failed

**Comment by rubenvp8510** [comment]
> Ahh, monolithic-receivers-tls failed

Flaky test

File: .chloggen/reencrypt_gw_default.yaml (hunk 2 above)
Comment (rubenvp8510) [reply .chloggen/reencrypt_gw_default.yaml:2]: I don't think this is a breaking change, as the other things will remaing working in the same way. I think this only adds a couple of hosts to the cert.

**Review by andreasgerstmayr — APPROVED** [review]

**Pull request merged**